### ISSUE TITLE
CLI tests not running by default

### ISSUE THREAD

**Issue by eriktaubeneck** [private-attribution/ipa#1275]
we probably need to open an issue to investigate - this shouldn't be required and all tests from inside `cli` module should be exercised as part of our in-memory suite (`cargo test`)

_Originally posted by @akoshelev in https://github.com/private-attribution/ipa/pull/1273#discussion_r1755895475_

**Comment by eriktaubeneck** [comment]
I looked into this a bit further, and these aren't running because they are gated behind features `cli` and `test-fixture`, which aren't default features.

The difference between `cargo test` and `cargo test --features="cli test-fixture" is the inclusion of 19 tests:
```
> test cli::crypto::tests::decrypt_bad_private_key - should panic ... ok
> test cli::crypto::tests::decrypt_no_enc_file - should panic ... ok
> test cli::crypto::tests::encrypt_and_decrypt ... ok
> test cli::crypto::tests::encrypt_and_execute_query ... ok
> test cli::crypto::tests::encrypt_bad_network_file - should panic ... ok
> test cli::crypto::tests::encrypt_incomplete_network_file - should panic ... ok
> test cli::crypto::tests::encrypt_no_network_file - should panic ... ok
> test cli::playbook::input::tests::from_str ... ok
> test cli::playbook::input::tests::input_source::multiline ... ok
> test cli::playbook::input::tests::parse_empty - should panic ... ok
> test cli::playbook::input::tests::parse_negative - should panic ... ok
> test cli::playbook::input::tests::tuple ... ok
> test cli::playbook::input::tests::tuple_parse_error - should panic ... ok
> test gen_events::tests::event_timestamp_arithmetics ... ok
> test gen_events::tests::event_timestamp_ordering ... ok
> test gen_events::tests::same_seed_generates_same_output ... ok
> test models::tests::event_timestamp_cmp ... ok
> test models::tests::event_timestamp_from ... ok
> test models::tests::event_timestamp_new ... ok
```

(I have the full output of each of these in files, if that's useful.) All tests run by `cargo test` are also run by `cargo test --features="cli test-fixture".

It seems like the best approach here is to simply update `pre-commit` and `.github/workflows/check.yml` to run `cargo test --features="cli test-fixture" instead of `cargo test`, as opposed to adding these as default features.

**Comment by eriktaubeneck** [comment]
I'll update #1273

**Comment by eriktaubeneck** [comment]
fixed in #1273. closing
